aws/templates/common.ftl

```diff
@@ -1891,61 +1891,9 @@
         } +
         attributeIfTrue("Instance", port.LB.Instance??, port.LB.Instance!"") +
         attributeIfTrue("Version",  port.LB.Version??, port.LB.Version!"") +
-        attributeIfContent("PortMapping",  portMapping)
+        attributeIfContent("PortMapping",  portMapping) + 
+        attributeIfContent("DestinationPort" , port.Name)
     ]
-
-    [#-- This lookup is purely to determine the correct attributes for the link   --]
-    [#-- The calling code should still check that the target link actually exists --]
-    [#-- using the value returned by this function                                --]
-    [#if targetTierId?has_content && targetComponentId?has_content]
-        [#local targetLoadBalancer = getLinkTarget(occurrence, targetLink) ]
-
-        [@cfDebug listMode targetLoadBalancer false /]
-
-        [#if targetLoadBalancer?has_content ]
-
-            [#local targetGroup = port.LB.TargetGroup]
-
-            [#if (ports[targetLoadBalancer.State.Attributes["PORT"]].Protocol) != "TCP" ]
-                [#local targetPath = port.LB.Path]
-            [#else]
-                [#local targetPath = "" ]
-            [/#if]
-
-            [#if targetPath?has_content]
-                [#-- target group name must be provided if path provided --]
-                [#if !targetGroup?has_content]
-                    [@cfException
-                        listMode "No target group for provided path" occurrence /]
-                    [#local targetGroup = "default" ]
-                [/#if]
-            [#else]
-                [#if !targetGroup?has_content]
```

Comment:
Good to see this logic go. It was always kind of clunky - moving all the tg stuff to the LB feels better.